appeals/web/src/server/appeals/appeal-details/appellant-case/application-outcome/application-outcome.mapper.js

```diff
@@ -14,48 +15,38 @@ export const changeApplicationOutcomePage = (appealData, appellantCaseData, stor
 
 	/** @type {PageContent} */
 	const pageContent = {
-		title: 'Change the application outcome',
+		title: 'Change application decision outcome',
 		backLinkUrl: `/appeals-service/appeal-details/${appealData.appealId}/appellant-case`,
 		preHeading: `Appeal ${shortAppealReference}`,
-		heading: `What was the application decision?`,
+		heading: `Change application decision outcome`,
 		pageComponents: [
-			{
-				type: 'radios',
-				parameters: {
-					name: 'application-outcome',
-					idPrefix: 'application-outcome',
-					fieldSet: {
-						legend: {
-							text: `What was the application decision?`,
-							isPageHeading: false,
-							classes: 'govuk-fieldset__legend--1'
-						}
+			radiosInput({
+				name: 'application-outcome',
+				id: 'application-outcome',
```

Comment:
we should stick with `idPrefix`. That better ensures unique ids for the items.
https://design-system.service.gov.uk/components/radios/

Reply:
Ah, I see the function takes in `id`. Either the function should be updated to take in idPrefix and move it all the way through, or just output idPrefix.

Reply:
Great spot, I was sure I'd fixed this before but apparently not! The function has been updated to taken `idPrefix` instead now, and all calling code updated accordingly.